scripts/blender_load_replay.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
```

Comment:
Let's rename to PR from `Added GFX Blender Replay Script` to `Added Blender Gfx-replay Script`. I want to be consistent. `GFX` is just an abbreviation for graphics and is too vague.